desktop/src/com/nerds/gamejam/desktop/DesktopLauncher.java

```diff
@@ -10,8 +10,9 @@
 	public static void main (String[] arg) {
 		Dimension screenSize = Toolkit.getDefaultToolkit().getScreenSize();
 		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
-		config.width = 4 * screenSize.width / 5;
-		config.height = 4 * screenSize.height / 5;
+		config.width = screenSize.width;
+		config.height = screenSize.height;
+		config.fullscreen = true;
```

Comment:
Add leave this class as is for time being as its a pain to work with when its full screen

Reply:
Cool, cool, I've put this back to the way it was